assets/javascripts/discourse/initializers/encrypt-drafts.js

```diff
@@ -61,17 +48,15 @@ export default {
           // TODO: https://github.com/emberjs/ember.js/issues/15291
           let { _super } = this;
 
-          if (
-            !globalContainer ||
-            globalContainer.isDestroyed ||
-            globalContainer.isDestroying
-          ) {
+          if (this.isDestroying || this.isDestroyed) {
```

Comment:
I don't think this will ever be true - we're in `modifyClassStatic` here, not `modifyClass`. So `this` will be the class, not an instance.